src/main/java/at/hannibal2/skyhanni/features/garden/AtmosphericFilterDisplay.kt

```diff
@@ -0,0 +1,61 @@
+package at.hannibal2.skyhanni.features.garden
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import at.hannibal2.skyhanni.utils.TimeUtils
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class AtmosphericFilterDisplay {
+
+    private val config get() = SkyHanniMod.feature.garden.atmosphericFilterDisplay
+
+    private var display = ""
+
+    private enum class Season(
```

Comment:
Maybe move this into a standalone class in the utils packet and move getSeasonByName inside this class' companion body

Reply:
done